src/test/fixtures/fixtures_test.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from 'chai';
+import * as chalk from 'chalk';
+import * as diff from 'diff';
+import * as fs from 'mz/fs';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+
+import exec from '../util/exec';
+
+// Install source map support for stack traces, etc.
+require('source-map-support').install();
```

Comment:
You can also put `--require source-map-support/register` as a line in `mocha.opts` to achieve this and it ensures it is loaded as early as possible so you can ensure proper .ts file references in places which might be loaded early; for example `../util/exec.ts` lines might not be mapped?  I'd have to check that bit ...

Reply:
Ha, you know what it's already in there. Removing this then...